specs/patterns.md

```diff
@@ -402,4 +402,149 @@ In this case you want to specify that A depends on B and C -- and that "installi
 
 ### Implementations
 
-None known.
\ No newline at end of file
+None known.
+
+
+## Document Data Resources
```

Comment:
I think "JSON Data Resources" would be more accurate than "Document Data Resouces" as "Document" is quite generic.

Reply:
yeah, I was wanting to leave it open to extension via multiple formats (like `edn`, `xml`), but thinking more they can't really be properly described by json schemas and would need a different resource spec anyways

Reply:
@micimize are you going to rename the section in line with that?